app/services/proofing/address_result.rb

```diff
@@ -0,0 +1,41 @@
+module Proofing
+  class AddressResult
+    attr_reader :success, :errors, :exception, :vendor_name, :transaction_id, :reference
+
+    def initialize(
+      success:,
+      errors:,
+      exception:,
+      vendor_name:,
+      transaction_id: '',
+      reference: ''
```

Comment:
maybe easier to default reference to nil than empty string?

Reply:
I'm not sure I understand the reasoning for making it nil vs empty string? I think all we do is log it and the result is the same in the end, right?

Reply:
I guess `nil` to me is a more real "isn't there" vs some sort of default from empty string, but you're right, for most situations they're the same